scripts/extract-ios-device-names.ts

```diff
@@ -81,37 +79,26 @@ const template = (contents: string) => {
   `;
 };
 
-const formatOutput = async (unformatted: string) => {
-  const config = await prettier.resolveConfig(outputPath);
-  if (config) {
-    return prettier.format(unformatted, {...config, parser: 'typescript'});
-  }
-
-  return unformatted;
-};
-
 export async function extractIOSDeviceNames() {
   const files = await getDefinitionFiles();
-  const definitions = collectDefinitions(files);
-  const formatted = await formatOutput(
-    template(JSON.stringify(definitions, undefined, 2))
-  );
+  const definitions = await collectDefinitions(files);
+  const formatted = template(JSON.stringify(definitions, undefined, 2));
```

Comment:
We need to sort the values by keys to have a predictable output (this was enforced by the sync code earlier), else we risk having constant changes due to the code being async and this showing up as a changed file

Reply:
The order is always preserved. We are running async code but we iterate one by one.